doc/CMakeLists.txt

```diff
@@ -46,6 +49,27 @@ if(DOXYGEN_FOUND AND NOT ${SPHINX_BUILD} STREQUAL SPHINX_BUILD-NOTFOUND)
         ${PROJECT_SOURCE_DIR}/modules/*.h
     )
 
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/first/first_example.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/first/first_example.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/material/mg_xs.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/material/mg_xs.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/ortho_2D.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/ortho_2D.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/ortho_3D.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/ortho_3D.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/read_2D_obj.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/read_2D_obj.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/transport_simulation_part_2D.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/transport_simulation_part_2D.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/extrusion.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/extrusion.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/transport_simulation_part_3D.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/transport_simulation_part_3D.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/read_3D_vtu.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/read_3D_vtu.md)
+    lua2md(${CMAKE_SOURCE_DIR}/doc/tutorials/lbs/meshing/transport_simulation_part_3D_single_agg.lua
+            ${CMAKE_BINARY_DIR}/doc/src/tutorials/lbs/meshing/transport_simulation_part_3D_single_agg.md)
```

Comment:
I would lean towards them being in `tutorials`, but am open for arguments otherwise. My logic is that it can be a standard place (in addition to test) for a user to grab a detailed input file to modify.

Reply:
I talked with @ragusa yesterday and we arrived to the conclusion, that we would want tutorials in `tutorials/` dir as well. But, there is a little bit more work to do for that to happen, so for right now, we would keep them in `doc/tutorials` and then the move would happen in an separate PR.

Reply:
The rst files will need to be moved. There will be another PR for this.